puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -0,0 +1,63 @@
+def databaseFile (id) {
+    writeFile(file: 'config/database.yml', text: """
+test:
+  adapter: postgresql
+  database: ${id}-test
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+
+development:
+  adapter: postgresql
+  database: ${id}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+
+production:
+  adapter: postgresql
+  database: ${id}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+""")
+
+}
+
+def addGem() {
+    sh 'echo "gemspec :path => \'../\', :development_group => :dev" >> bundler.d/Gemfile.local.rb'
```

Comment:
Could this be a `writeFile` to `bundler.d/plugin.local.rb` (or whatever else name), so that we don't need an ugly `>>` to append stuff to an existing file?